### DISCUSSION TITLE
Formulations from the previous assignments

### DISCUSSION THREAD

**Discussion by frichterTH** [irgroup-classrooms/dis17-2021#37] [Q&A] [answered]
Is it allowed to take formulations from the previous assignments one-to-one or do they need to be reworded?

**Comment by MartinBilko** [comment]
@phschaer

**Comment by phschaer** [comment, marked as answer]
It is okay to take over *parts* of the previous assignment, but only parts, and please indicate which parts you took over. It's the same procedure as when citing others' work and literature. Otherwise, this would be (self-)plagiarism.

**Comment by ghost** [comment]
Hallo Herr Schaer,

wir sind etwas ratlos bezüglich Ihrer Antwort - die Aufgabenstellung für Assignment III verlangt ja inhaltlich exakt dieselben Punkte wie Assignment IV. Wir hatten deswegen unser Paper explizit schon so angelegt, dass wir es nur noch erweitern müssen bzw. dass es eine Art Abstract des finalen Papers ist, auch schon mit Quellen und Abbildungen etc. Technisch wurde ja nach der Präsentation auch nichts mehr an der eigentlichen Aufgabe verändert oder ergänzt. Inwiefern ist das dann nicht legitim, den bestehenden Text (weiter) zu verwenden bzw. welche weiteren Inhalte (außer ausführlicherer Darstellung) erwarten Sie?

Viele Grüße
Helga Wolf
für die Gruppe TheGitGangTheory

**Reply by phschaer** [reply]
Kein Problem - Es geht mir nur darum, dass nicht einfach nochmal einen 1:1-Kopie des Zwischenberichtes eingereicht werden, sondern dass schon ein Plus zu erkennen ist. Solange das transparent gemacht wird, ist alles gut. Dann können Sie gerne auch Text-Teile, Tabellen, Abbildungen aus der vorherigen Version übernehmen. Solange Sie z.B. schreiben: "Der Folgende Abschnitt entspricht dem Abschnitt XYZ des Zwischenberichtes und wurde um eine Analyse der Ergebnisse auf Seite 3 erweitert..." - Oder so etwas in der Art, ist alles gut. So wie beim Zitieren es ja auch okay ist, die Gedanken, Ideen oder auch Textteile komplett zu übernehmen, so lange halt klar und ersichtlich ist, woher das jeweils stammt. 

Ist es nun klarer, was ich meine?

Alles Gute

Philipp Schaer

**Comment by ghost** [comment]
ja, prima. vielen dank!

VG
helga wolf
